Makefile

```diff
@@ -109,7 +109,11 @@ upgrade: ## Upgrade dependencies
 .PHONY: checkgenerate
 checkgenerate:
 	@# Used in CI to verify that `make generate` doesn't produce a diff.
-	test -z "$$(git status --porcelain | tee /dev/stderr)"
+	@echo git status --porcelain
+	@if [[ -n "$$(git status --porcelain | tee /dev/stderr)" ]]; then \
+	  git diff; \
```

Comment:
Why did this need to change?

Reply:
This was so that "checkgenerate" failures are more helpful.

This check failed in 2a3a31bf671711bab42646ac523455f51f6c04cc. The output only showed the file names. I was unable to reproduce it locally (turned out to be my bad), so I thought it would be more helpful if CI actually showed the changes that occured.

So when there is an issue with the "checkgenerate" task, this now shows the whole diff.